### PR TITLE
chore: schema docs, add to all examples

### PR DESCRIPTION
**Pull request by toddbaert**
I've added a small page about the schema, and I've added the schema to all examples.

Preview: https://deploy-preview-1181--polite-licorice-3db33c.netlify.app/reference/schema/

### CODE REVIEW

File: docs/reference/specifications/protos.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Flag evaluation API
 
 This proto forms the basis of a flag-evaluation API.
 It supports single and bulk evaluation RPCs, and flags of various types, as well as establishing a stream for getting notifications about changes in a flag definition.
-It supports the inclusion of a &#34;context&#34; with each evaluation, which may contain arbitraty attributes relevant to flag evaluation.
+It supports the inclusion of a &#34;context&#34; with each evaluation, which may contain arbitrary attributes relevant to flag evaluation.
```
Comment (beeme1mr) [inline docs/reference/specifications/protos.md:11]: Isn't this generated?

**Review by beeme1mr — APPROVED** [review]

File: docs/reference/specifications/protos.md (hunk 1 above)
Comment (toddbaert) [reply docs/reference/specifications/protos.md:11]: Yep. Sorry, I don't even remember editing this. Must have been pending or automatic.

Comment (toddbaert) [reply docs/reference/specifications/protos.md:11]: I guess it doesn't hurt to keep it for now, I'll fix it upstream.

Comment (toddbaert) [reply docs/reference/specifications/protos.md:11]: Actually it's already fixed upstream... that must be why this magically appeared.

**Pull request merged**